src/hub/interfaces/IHubRegistry.sol

```diff
@@ -11,8 +11,9 @@ interface IHubRegistry is IERC6909Decimals {
     event NewPool(PoolId poolId, address indexed manager, AssetId indexed currency);
     event UpdateManager(PoolId indexed poolId, address indexed manager, bool canManage);
     event SetMetadata(PoolId indexed poolId, bytes metadata);
-    event UpdateDependency(bytes32 indexed what, address dependency);
+    event UpdateDependency(PoolId indexed poolId, bytes32 indexed what, address dependency);
```

Comment:
I originally changed dependencies to be per pool, to use this to store the hub request manager. Later decided to introduce separate new storage. But I still feel having dependencies be indexed per pool makes sense, for more flexibilty.